pkg/cluster/internal/loadbalancer/config.go

```diff
@@ -45,6 +48,8 @@ defaults
   timeout connect 5000
   timeout client 50000
   timeout server 50000
+  # allow to boot despite dns don't resolve backends
```

Comment:
in the future we might want to clarify this comment a bit, something like "don't exit when we can't yet resolve backends"